### ISSUE TITLE
GetInterfaceIP Does not correctly handle IPv6 addresses. 

### ISSUE THREAD

**Issue by MaddHatters** [salbahra/cordova-plugin-networkinterface#44]
https://github.com/salbahra/cordova-plugin-networkinterface/blob/e558143dc373c4192422682b8a56d5503ea62b2e/src/ios/CDVNetworkInterface.m#L18

Recommend checking only for `sa_type == AF_INET` as we cannot reliably convert all IPv6 addresses to IPv4 (Given there are more IPv6 addresses available than IPv4). We could make a new method specifically for IPv6 maybe?

In my testing inet_ntoa converts IPv6 to `0.0.0.0`

**Comment by ZaLiTHkA** [comment]
Some of my app users are running into the same problem, with my app reporting their IP address as `0.0.0.0`. Knowing this, at least now I can show a more useful message to the users when this comes up..

**Comment by MaddHatters** [comment]
@ZaLiTHkA I actually fixed this issue and forked the repo if you want to use that (https://github.com/MaddHatters/cordova-plugin-networkinterface)

To use the fork in your repo I think you can use one of the following:

```
npm uninstall cordova-plugin-networkinterface
npm install MaddHatters/cordova-plugin-networkinterface
or
npm install git+https://git@github.com/MaddHatters/cordova-plugin-networkinterface
```

After you install that, I believe you will need to clear out the plugin from the plugins folder and possibly wipe out the platform you are building for too to get the changes on the next build. I hope that helps! Let me know if you run into any trouble.


I have a pending PR into the main branch as well but that may never be merged in.

**Comment by ZaLiTHkA** [comment]
@MaddHatters that's awesome, thanks for letting me know.. I'm no stranger to changing out Cordova plugins, so I'll give your fork a try just now.

**Comment by MaddHatters** [comment]
@ZaLiTHkA You're welcome! I'd love to hear if it works for you!

**Comment by salbahra** [comment]
This should be fixed in #45. Thank you @MaddHatters

**Comment by ZaLiTHkA** [comment]
my boss gave me a bunch of other work to do last week, so I only managed to get back to this one today.

had a look at the change merged in #45, if I'm reading this correctly, it looks like the plugin will now simply ignore IPv6 addresses, now returning `null` instead of the `0.0.0.0` that it did previously.. is that correct?

**Comment by MaddHatters** [comment]
Yes, that is correct. I check for the existence of the address object and
the IP on my end.
